scenario/context.py

```diff
@@ -409,6 +388,16 @@ def test_foo():
         (local_path / 'foo' / 'bar.yaml').write_text('foo: bar')
         scenario.Context(... charm_root=virtual_root).run(...)
 
+    The class can also provide a context manager, like::
```

Comment:
```suggestion
    The class can also be used as a context manager, like::
```

Reply:
The `Context` class isn't really the context manager, though, right? The manager is the object that has the `__enter__` and `__exit__`, which is the `_Manager` class in this case. But then maybe that's just an implementation detail that users shouldn't care about, so "be used as" is clearer even though not 100% accurate?

Maybe better to avoid "context manager" entirely and have something like this?

```suggestion
    If you need access to the charm object that will handle the event, use the class in a with statement, like::
```

Reply:
I guess my reasoning above is also an argument for going back to `mgr` or rather than `event` or `hook` as the example name. I liked `event` most when I had the variant that did `event.emit()`. I guess in some ways it's like an `ops._Manager`, so that also leans towards `mgr` as the recommended name.

Reply:
mmm yeah I guess you're right that Context isn't technically speaking the context manager, but I feel we could get away with it :) 
weak preference though, so feel free to go your way with this.